.github/workflows/publish.yml

```diff
@@ -22,10 +21,12 @@ jobs:
         with:
           # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
           fetch-depth: 0
-      - name: Set up JDK 8
-        uses: olafurpg/setup-scala@v10
+
+      - name: Set up JDK 11
```

Comment:
```suggestion
      - name: Set up JDK 8
```

Reply:
you just scary me! That's just the title. Ufa!

Reply:
Yep, just for syncing the title and real job 😉